bitstore/services.py

```diff
@@ -5,6 +5,12 @@
 
 _public_key = None
 
+# FileManager
+from filemanager.models import FileManager
```

Comment:
why not with the imports?

Reply:
@akariv not sure what you mean here

Reply:
Why is this import not grouped with the other imports